### DISCUSSION TITLE
eslint v9 is a war crime

### DISCUSSION THREAD

**Discussion by HyperCrowd** [typescript-eslint/typescript-eslint#9334] [Community Feedback]
The dependency hell that eslint v9 is causing this project is a war crime

I hope you guys are getting compensated some how, because right now, this plug-in is complete unusable because of their stupid breaking changes

I am on hour 20 of trying to get this plugin to work and it's just not possible

I'm at the point where my bash script is just brute forcing every combination of `8.0.0-alpha.x` just to see which combination of numbers doesn't explode `npm install`

**Comment by JoshuaKGoldberg** [comment]
Hi @HyperCrowd, sorry for the delay responding here. It seems like your feedback is a combination of:

* Venting about ESLint core's v9 rollout (and maybe also flat config)
* Suggesting that our `8.0.0-alpha.x` versioning is hard to work with as a result
* Commiserating about the work we in typescript-eslint have had to go through

In order...

> venting

If you have constructive feedback about the ESLint v9 rollout, I'd encourage you to go to the ESLint project itself. We're a separate team from ESLint and can't do anything with that feedback. 

Please keep in mind that ESLint, typescript-eslint, and many other open source projects are largely run by volunteers and/or independent open source maintainers. We're not a for-profit organization with lots of funding. We're doing the best we can with extremely limited resources to support a massive, ever-growing ecosystem. Hyperbolizing a rollout as _"a war crime"_ is neither helpful nor appropriate. Especially given that many folks in and out of open source are actively impacted by _actual war crimes_ happening in the world today.

> our `8.0.0-alpha.x` versioning

If there's anything you think we could do to better support you or other users, such as improving docs or rethinking our rollout strategy, please do let us know in a dedicated issue, discussion, or Discord thread as appropriate per our [contributing guidelines](https://typescript-eslint.io/contributing).

> Commiserating

Thank you 🙂 we do receive some funding on https://opencollective.com/typescript-eslint. I'd encourage anybody who wants to see more progress being done on linting to fund us, https://opencollective.com/eslint, and other projects they depend on.

---

We've locked this thread as not constructive. But, again, if you have something to say that can improve typescript-eslint, please go through our contributing guidelines and post. Thanks! ❤️
